packages/PyTrilinos/cmake/PyTrilinos_config.h.in

```diff
@@ -53,14 +53,17 @@
 /* Define if Domi is enabled */
 #cmakedefine HAVE_DOMI
 
+/* Define if EpetraExt ModelEvaluator is flagged to be built */
+#cmakedefine EpetraExt_BUILD_MODEL_EVALUATOR
+
 /* Define if Epetra namespace is enabled within NOX */
-#cmakedefine HAVE_NOX_EPETRA
+#cmakedefine NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA
```

Comment:
I'm not keen on this macro being defined in a PyTrilinos header file.  Doesn't this more properly belong in NOX?  If NOX decides to define this macro, you'll have a double definition.  Alternately, you could add a CMake option `PyTrilinos_NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA` and a corresponding PyTrilinos macro `PYTRILINOS_NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA`.

Reply:
OK, I have changed this, but I have a lot of inconsistency in defining configuration macros for PyTrilinos. I am going to open up a new issue for dealing with that. For now, I no longer define `NOX_ENABLE_ABSTRACT_IMPLEMENTATION_*` in PyTrilinos header files.